README.md

```diff
@@ -7,7 +7,8 @@ It relies on the presence of a `last-dd-base` tag in the branch to find your lat
 
 ## Usage
 
-1. Add a secret called `WORKFLOW_TOKEN` with write access to your repository
+1. Add a secret called `WORKFLOW_TOKEN` with write access to your repository. This must be a github access token with at least the following permissions (the scope to update the Github Actions is needed in case the upstream repo modify its own actions):
```

Comment:
```suggestion
1. Add a secret called `WORKFLOW_TOKEN` with write access to your repository. This must be a [personal github access token](https://github.com/settings/tokens) with at least the `repo` and `workflow` permissions (the scope to update the Github Actions is needed in case the upstream repo modifies its own actions):
```

Maybe like so? Not sure the perms screenshot is needed

Reply:
And: Authorize SSO for the Datadog org on the token